pkg/core/tekton/tekton.go

```diff
@@ -454,3 +459,19 @@ func getInputCodesetPath(inputs []*workflow.WorkflowStepInput) string {
 	}
 	return ""
 }
+
+func (w *WorkflowBackend) tektonDeleteIfError(ctx context.Context, log *log.Logger, err *error, tektonWorkload interface{}) {
+	if *err != nil {
+		switch tw := tektonWorkload.(type) {
+		case *v1alpha1.TriggerTemplate:
+			log.Printf("Deleting TriggerTemplate: %s...", tw.Name)
```

Comment:
Just one nit-pick: could you extend the error message so that it's clear you're deleting it _because of an error during creation_?

Reply:
yes, makes sense